<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="coatChange" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="usedCardDeck" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>140</height>
     </rect>
    </property>
    <widget class="QLabel" name="discardVisual">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="cardDeck" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>100</width>
      <height>140</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="BetWidget" name="betZone" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QLabel" name="betLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="balanceLabel">
     <property name="geometry">
      <rect>
       <x>138</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="standButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>240</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hitButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>300</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Deal/ play</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameResult">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="CardWidget" name="playerHand" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>383</width>
      <height>103</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dealerScore">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>73</y>
      <width>229</width>
      <height>46</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="CardWidget" name="dealerHand" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>229</width>
      <height>47</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CardWidget</class>
   <extends>QWidget</extends>
   <header>cardwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BetWidget</class>
   <extends>QWidget</extends>
   <header>betwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
